addons/account/models/account_reconcile_model.py

```diff
@@ -128,7 +128,7 @@ class AccountReconcileModel(models.Model):
 
     # Base fields.
     active = fields.Boolean(default=True)
-    name = fields.Char(string='Name', required=True)
+    name = fields.Char(string='Name', required=True, translate=True)
```

Comment:
it doesn't seem that we can do this in stable (if I do this without "-u account", I get database error because we do JSON operator over a column that is not of JSONB type)